README.md

```diff
@@ -146,3 +146,23 @@ var playerParam = {
 
 ### Advanced Customization
 Advanced customization is readily available by modifying JS files. Follow [Developer Setup](#developer-setup) section to create a local repository and to run build script. Built files are available inside build folder. You are welcomed to host your built skin javascript to be run with your player.
+
+##How to Contribute
+
+If you send a pull request, please do it against the [`master` branch](https://github.com/ooyala/html5-skin/tree/master). We maintain stable branches separately but we don't accept pull requests to them directly. 
+
+We will do our best to keep the `master` branch in good shape, with tests passing at all times. But our `master` branch is under active development and may contain changes that your application might not be compatible. We recommend using [latest stable version](https://github.com/ooyala/html5-skin/tree/stable) for production.
```

Comment:
funny thing... I removed 'with' because I was taught to never end a sentence with a preposition. [Quick research](https://www.grammarly.com/blog/youve-been-lied-to-heres-why-you-absolutely-can-end-a-sentence-with-a-preposition/) as reveled this is only for grammar snobs and it is in fact acceptable to end a sentence with a preposition.